lte/gateway/c/session_manager/CMakeLists.txt

```diff
@@ -54,7 +55,7 @@ endif ()
 
 include_directories("${OUTPUT_DIR}")
 include_directories("${MAGMA_ROOT}/orc8r/gateway/c/common/logging")
-
+include_directories("${MAGMA_ROOT}/lte/gateway/c/oai/lib/mobility_client")
```

Comment:
We definitely do not want a dependency into `lte/gateway/c/oai/lib/mobility_client`. Let's create a mobilityd client in SessionD if we need to talk to mobilityd

Reply:
Yes Marie, I already fixed same things in magma-5G. I will add here.

Reply:
Changes done.